hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestMetaRegionReplicaReplication.java

```diff
@@ -385,6 +386,12 @@ private void primaryNoChangeReplicaIncrease(final long[] before, final long[] af
     }
   }
 
+  private void PrimaryIncreaseReplicaIncrease(final long[] before, final long[] after) {
```

Comment:
the leading char should be lower case :)

Reply:
Done.